test/Microsoft.NET.Build.Tests/GivenThatWeWantToSetPropertiesInDirectoryBuildProps.cs

```diff
@@ -0,0 +1,82 @@
+﻿using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Microsoft.NET.TestFramework.Commands;
+using static Microsoft.NET.TestFramework.Commands.MSBuildTest;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml.Linq;
+using Xunit;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToSetPropertiesInDirectoryBuildProps : SdkTest
+    {
+
+        [Fact]
+        public void The_default_configuration_can_be_set_to_release()
+        {
+            TestProject project = new TestProject()
+            {
+                Name = "DirectoryBuildPropsTest",
+                TargetFrameworks = "netstandard1.4",
+                IsSdkProject = true
+            };
+
+            var testAsset = _testAssetsManager.CreateTestProject(project);
+
+            string directoryBuildPropsPath = Path.Combine(testAsset.Path, "Directory.Build.props");
+            
+            var directoryBuildPropsContent = @"
+<Project>
+  <PropertyGroup>
+    <Configuration Condition="" '$(Configuration)' == '' "">Release</Configuration>
```

Comment:
`Optimize` and `DebugSymbols` are defaulted based on whether the configuration is Debug or Release.  The original issue leading to this change was that they wouldn't be set correctly if you set the Configuration in Directory.Build.props, so that's what I'm testing here.